README.md

```diff
@@ -274,77 +274,147 @@ Response (Un-authenticated)
-          }
+        "id": "-PGe3UUH4wsDg0ttqc7G-",
+        "visibility": ["friends"],
+        "primary": {
+          "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+          "width": 2000,
+          "height": 1500
         },
-        "emoji": "😍",
-        "type": "heartEyes",
-        "uri": "https://storage.bere.al/Photos/<>/realmoji/<>-realmoji-heartEyes-1659046828.jpg",
-        "date": {
-          "_seconds": 1660002563,
-          "_nanoseconds": 350000000
-        }
+        "secondary": {
+          "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+          "width": 2000,
+          "height": 1500
+        },
+        "caption": "I love my tortilla blanky 🥰",
+        "retakeCounter": 0,
+        "lateInSeconds": 16110,
+        "isLate": true,
+        "isMain": true,
+        "realMojis": [
+          {
+            "id": "Jn8tDy3wwwJ197BHr7lJs",
+            "user": {
+              "id": "[REDACTED]",
+              "username": "[REDACTED]",
+              "profilePicture": {
+                "url": "https://cdn.bereal.network/Photos/[REDACTED]/profile/[REDACTED]",
+                "width": 1000,
+                "height": 1000
+              }
+            },
+            "media": {
+              "url": "https://cdn.bereal.network/Photos/[REDACTED]/realmoji/[REDACTED]",
+              "width": 500,
+              "height": 500
+            },
+            "type": "up",
+            "emoji": "👍",
+            "isInstant": false,
+            "postedAt": "2023-07-14T02:47:15.850Z"
+          }
+        ],
+        "comments": [],
+        "unblurCount": 0,
+        "takenAt": "2023-07-14T02:46:26.520Z",
+        "creationDate": "2023-07-14T02:46:26.657Z",
+        "updatedAt": "2023-07-14T02:46:26.657Z"
       }
-    ],
-    "screenshots": [],
-    "screenshotsV2": []
+    ]
   },
-]
+  "friendsPosts": [
+    {
+      "user": {
+        "id": "[REDACTED]",
+        "username": "[REDACTED]",
+        "profilePicture": {
+          "url": "https://cdn.bereal.network/Photos/[REDACTED]/profile/[REDACTED]",
+          "width": 1000,
+          "height": 1000
+        }
+      },
+      "momentId": "L1O8S_Tbph2fpEeuoSWcv",
+      "region": "us-central",
+      "moment": {
+        "id": "L1O8S_Tbph2fpEeuoSWcv",
+        "region": "us-central"
+      },
+      "posts": [
+        {
+          "id": "HLAxwJatn4OZFmLieuM8N",
+          "primary": {
+            "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+            "width": 1500,
+            "height": 2000
+          },
+          "secondary": {
+            "url": "https://cdn.bereal.network/Photos/[REDACTED]/post/[REDACTED]",
+            "width": 1500,
+            "height": 2000
+          },
+          "location": {
+            "latitude": 43.079,
+            "longitude": -123.5797
```

Comment:
I have checked and this is just on a highway, so not too worried about exposing someones location here

Reply:
Thanks for that. Totally forgot about location data. 🤦‍♂️